src/components/WelcomeView.jsx

```diff
@@ -89,6 +84,8 @@ const StyledLogoGrey = styled(LogoGrey)({
 });
 
 function WelcomeView({ terraUrl }) {
+  const theme = useTheme();
+
```

Comment:
```suggestion

```
Why do we need to call useTheme()? The custom theme should be piped through from the index page.